cypress/e2e/undo-redo/block-configs-should-update-flow-state/open-response-configs.cy.js

```diff
@@ -1,115 +0,0 @@
-describe('non common configs on open response block and apply undo/redo to changes', () => {
-    beforeEach(() => {
-    cy.createFlow({
-    label: 'Undo/Redo open response configs ',
-    })
- })
- it('should configure and test the block', () => {
-    const blockConfigs = {
-        label: 'your name',
-            name: {
-                beforeEdit: 'your_name',
-                afterEditPostFix: '_updated',
-            },
-            resources: {
-                ivr: ' 02_flowers_for_albert.mp3',
-                sms: 'Kindly enter your name',
-                ussd: 'Kindly enter your name',
-            },
-        
-        duration: 
-        {
-           maxDuration:5,
```

Comment:
no need to retest common configs @pascaleli , we'd just focus on those 02 specific configs for this block